mesheryctl/internal/cli/root/root.go

```diff
@@ -110,6 +110,7 @@ func init() {
 
 	availableSubcommands = []*cobra.Command{
 		versionCmd,
+		completionCmd,
```

Comment:
@WLun001 thank you for this PR. 👍 

We'll want to move the `completion` subcommand underneath the `system` command (e.g. `mesheryctl system completion`). `system` being a collection of commands pertaining to the management of Meshery itself, while `perf` and `mesh` being commands to manage service meshes.